docs/design-deployment.rst

```diff
@@ -44,6 +45,7 @@ between the components are represented in the following diagram:
    duplicator -> collector;
    duplicator -> recorder;
    stats -> exporter;
+   duplicator -> receiver1 -> receiver2;
```

Comment:
```suggestion
   duplicator -> receiver;
```

I'm not totally sure whether we want to add this here, since it's an alternative to the architecture expressed in the diagram...maybe we want to have two diagrams, or draw a box labeled "receiver" around both stats and collector.

Reply:
I'll try the box labeled receiver